app/app-version.gradle

```diff
@@ -0,0 +1,6 @@
+android {
+    defaultConfig {
+        versionCode 100000000
```

Comment:
What do you think about calculating `versionCode` and building `versionName` based on 3 extracted values:
```
def versionMajor = 1
def versionMinor = 0
def versionPatch = 0

 // ...

versionCode "$versionMajor.$versionMinor.$versionPatch"
versionName versionMajor * 100000000 + versionMinor * 10000 + versionPatch
```